cmd/create/cluster/cmd.go

```diff
@@ -3553,6 +3606,116 @@ func durationStringValidator(val interface{}) error {
 
 }
 
+// ------------------------ GPULimit Processing Helpers ------------------------
+
+func convertGPULimitsStringArrayToGPULimits(input [][]string) ([]GPULimit, error) {
+	gpuLimits := []GPULimit{}
+
+	// no input, ie assume value will not be set
+	if len(input) == 0 {
+		return []GPULimit{}, nil
+	}
+	// gpuLimits = append(gpuLimits, convertArrayToGPULimit(input[0]))
+	for index := range input {
+		tempGPULimit, err := convertArrayToGPULimit(input[index])
+		if err != nil {
+			return []GPULimit{}, fmt.Errorf("Failed to convert GPULimit %v from string to struct of type GPULimit, error: %s", index, err)
+		}
+		gpuLimits = append(gpuLimits, tempGPULimit)
+
+	}
+	return gpuLimits, nil
+}
+
+func convertArrayToGPULimit(input []string) (GPULimit, error) {
+	gpuMin, err := strconv.Atoi(fmt.Sprintf("%v", input[1]))
+	if err != nil {
+		return GPULimit{}, err
+	}
+	gpuMax, err := strconv.Atoi(fmt.Sprintf("%v", input[2]))
+	if err != nil {
+		return GPULimit{}, err
+	}
+	GPULimit := GPULimit{input[0], gpuMin, gpuMax}
+	return GPULimit, nil
+}
+
+func processTwoGPULimits(sanatizedInput string, splitIndex int) ([][]string, error) {
+	gpuLimitsOne := strings.Split(sanatizedInput[0:splitIndex], ",")
+	if len(gpuLimitsOne) != 3 {
+		return [][]string{}, fmt.Errorf("each gpuLimit should only have 3 values")
+	}
+	gpuLimitsTwo := strings.Split(sanatizedInput[splitIndex+3:len(sanatizedInput)-1], ",")
+	if len(gpuLimitsTwo) != 3 {
+		return [][]string{}, fmt.Errorf("each gpuLimit should only have 3 values")
+	}
+	var gpuLimits [][]string
+	gpuLimits = append(gpuLimits, gpuLimitsOne)
+	gpuLimits = append(gpuLimits, gpuLimitsTwo)
+	return gpuLimits, nil
+}
+
+func processOneGpuLimit(sanatizedInput string) ([][]string, error) {
+	gpuLimitsOne := strings.Split(sanatizedInput[0:len(sanatizedInput)-2], ",")
+	if len(gpuLimitsOne) != 3 {
+		return [][]string{}, fmt.Errorf("each gpuLimit should only have 3 values")
+	}
+	var gpuLimits [][]string
+	gpuLimits[0] = gpuLimitsOne
+	return gpuLimits, nil
+}
+
+func splitStringIntoGPULimitsArray(input string) ([][]string, error) {
```

Comment:
maybe not critical, but maybe a format like this will be easier to parse?
```nvidia.com/gpu,0,1 amd.com/gpu,4,16```

then we just do ``strings.Split(input, " ")`` to get the outer-scoped list and ``strings.Split(item, ",")`` to get the list of ``<type>,<min>,<max>``

Reply:
Open to this idea, however I think its more user friendly for them to write something that looks like an array of arrays, rather them to understand that those 6 values correspond to 2 gpuLimits with values of type, min and max. As well all the additional work has been done already to parse it so it doesn't add and overhead.